db/db.js

```diff
@@ -0,0 +1,28 @@
+const Sequelize = require("sequelize");
+const pkg = require('../package.json')
+
+const databaseName =
+    pkg.name + (process.env.NODE_ENV === 'test' ? '-test': '');
```

Comment:
Line 4-6: What was the test part of it for? If we don't have a separate test db, we could delete.